src/test/java/GameBrowserTest.java

```diff
@@ -0,0 +1,101 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class GameBrowserTest {
+
+    @Test
+    public void constructorTest() {
+        GameBrowser gameBrowser;
+
+        // Empty constructor use (invalid)
+        try {
+            gameBrowser = new GameBrowser();
+            fail("Using empty constructor should fail test.");
+        } catch(Exception e) {
+            assertEquals("java.lang.Exception", e.getClass().getName());
+            assertEquals( "Invalid constructor. Supply a data source file.", e.getMessage());
+        }
+
+        // Default constructor with empty non-existent file path (invalid)
+        assertThrows(IllegalArgumentException.class, () -> new GameBrowser(""));
+
+        // Default constructor with non-existent file path (invalid)
```

Comment:
Not using the default constructor

Reply:
@Rmelikyan This is here to ensure that it isn't used.